app/helpers/kaui/date_helper.rb

```diff
@@ -22,5 +22,25 @@ def format_date(date, timezone)
     def truncate_millis(date_s)
       DateTime.parse(date_s).strftime('%FT%T')
     end
+
+    def current_time(time_zone=nil)
+      time_zone = @account.time_zone unless @account.nil? || time_zone.nil?
+
+      # if time zone is not found return the time as is
+      return Time.now if time_zone.nil?
+
+      current_utc_time = nil
+      begin
+        # fetch time from killbill server
+        clock = Kaui::Admin.get_clock(time_zone, Kaui.current_tenant_user_options(current_user, session))
+        current_utc_time = clock['currentUtcTime']
+      rescue
```

Comment:
It works because Kill Bill errors extend `StandardError` (https://robots.thoughtbot.com/rescue-standarderror-not-exception#the-default-is-standarderror), but catching explicitly `KillBillClient::API::NotFound` as it was done previously would be cleaner.

For testing, we should either mock the server calls or add proper integration tests (requiring Kill Bill). Alternatively, we could also look at setting up VCR (https://github.com/vcr/vcr).